watcher/lib/constants/constants.go

```diff
@@ -171,14 +196,23 @@ var (
 		RetentionMedium: IntervalMedium,
 	}
 
-	// AllFunctions contains names of functions, excluding percentile (because percentile is
-	// formatted like 'percentile_X')
+	// AllFunctions contains names of functions,
+	// excluding percentile, bottom, sample and top
+	// because those functions needs an additional paramenter (percentile, 99)
```

Comment:
needs -> need

Reply:
that's already fixed in the latest commit.